### ISSUE TITLE
I am not able to simulate media keys

### ISSUE THREAD

**Issue by ShivamJoker** [moses-palmer/pynput#190]
I saw a pull request on vk media keys support but when i type those with
```
from pynput.keyboard import Key, Controller

keyboard = Controller()

# Press and release play pause
keyboard.press(Key.media_play_pause)
keyboard.release(Key.media_play_pause)
```
it's giving me error
```
Traceback (most recent call last):
  File "presskeys.py", line 6, in <module>
    keyboard.press(Key.media_play_pause)
  File "C:\Users\shivam\AppData\Local\Programs\Python\Python37\lib\enum.py", line 349, in __getattr__
    raise AttributeError(name) from None
AttributeError: media_play_pause
```

**Comment by moses-palmer** [comment]
Thank you for your report.

The reason is that the PR has not been merged. I am very hesitant to add functionality to only one platform, so merging would require the keys to be added to the other supported platform as well.

**Comment by ShivamJoker** [comment]
But why can't you implement it for all the OS ?
for simulating mac media keys here is the code
```
#!/usr/bin/python

import Quartz

# NSEvent.h
NSSystemDefined = 14

# hidsystem/ev_keymap.h
NX_KEYTYPE_SOUND_UP = 0
NX_KEYTYPE_SOUND_DOWN = 1
NX_KEYTYPE_PLAY = 16
NX_KEYTYPE_NEXT = 17
NX_KEYTYPE_PREVIOUS = 18
NX_KEYTYPE_FAST = 19
NX_KEYTYPE_REWIND = 20


def HIDPostAuxKey(key):
    def doKey(down):
        ev = Quartz.NSEvent.otherEventWithType_location_modifierFlags_timestamp_windowNumber_context_subtype_data1_data2_(
            NSSystemDefined,  # type
            (0, 0),  # location
            0xa00 if down else 0xb00,  # flags
            0,  # timestamp
            0,  # window
            0,  # ctx
            8,  # subtype
            (key << 16) | ((0xa if down else 0xb) << 8),  # data1
            -1  # data2
        )
        cev = ev.CGEvent()
        Quartz.CGEventPost(0, cev)
    doKey(True)
    doKey(False)


for _ in range(10):
    HIDPostAuxKey(NX_KEYTYPE_SOUND_DOWN)

HIDPostAuxKey(NX_KEYTYPE_PLAY)
```

**Comment by moses-palmer** [comment]
I have pushed a work-in-progress branch to [feature-media-keys](https://github.com/moses-palmer/pynput/tree/feature-media-keys).

Please note that I have not performed any tests yet; I do not have access to a _macOS_ system, my _Linux_ system runs _Wayland_ under which _pynput_ does not work, and my _Windows_ virtual machine does not seem to receive any media keys. I would very much appreciate your testing!

**Comment by moses-palmer** [comment]
I have merged these changes and released _pynput 1.5.0_.
